osu.Game/Rulesets/Difficulty/DifficultyCalculator.cs

```diff
@@ -87,11 +86,19 @@ private DifficultyAttributes calculate(IBeatmap beatmap, Mod[] mods, double cloc
 
                     currentSectionEnd += sectionLength;
                 }
-
                 foreach (Skill s in skills)
-                    s.Process(h);
+                {
+                    if (counted > 1)
```

Comment:
Instead of this, create a virtual method in `Skill` called `ComputeStrain()`. Have `Skill.Process()` call this method, and have `TouchSkill` derive this method with your implementation.

Then you'd have your `TouchSpeed` and `TouchAim` skills derive `TouchSkill`.

With that being said, I'm like 97% sure this sort of overriding is definitely _not_ what we want, and this looks like throwing darts at the wall. It doesn't make logical sense to me, and the absolute necessity of it is not described anywhere.

@Tom94 please see if you agree with my assertion.

Reply:
Issue has been resolved as a result of reverting strain calculation completely.